app/public/fullcalendar-5.5.1/lib/locales/af.js

```diff
@@ -0,0 +1,27 @@
+FullCalendar.globalLocales.push(function () {
```

Comment:
I agree with Elias here, it looks like there is a locale-all.min.js, let's keep that one file, and remove the other locales and see if the code still work as expected